jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -2720,9 +2720,17 @@ public String toLowerCase(Locale locale) {
 		}
 
 		if (StrHWAvailable() && language == "en") { //$NON-NLS-1$
-			String output = new String(lengthInternal());
-			if (toLowerHWOptimized(output))
-				return output;
+			int sLength = lengthInternal();
+
+			if (enableCompression && (null == compressionFlag || coder == LATIN1)) {
+				byte[] output = new byte[sLength];
+				if (toLowerHWOptimizedLatin1(value, output, sLength))
+						return new String(output, LATIN1);
+			} else {
+				byte[] output = new byte[sLength * 2];
+				if (toLowerHWOptimizedUTF16(value, output, sLength * 2))
+						return new String(output, UTF16);
```

Comment:
Let's stay consistent with the rest of the codebase and add curly brackets around if blocks.